presidio-analyzer/conf/analyzer.yaml

```diff
@@ -0,0 +1,93 @@
+recognizer_registry:
+  recognizers: 
+    - name: CreditCardRecognizer
+      supported_languages: 
+      - en
+      - es
+      type: predefined
+
+    - ItFiscalCodeRecognizer
+
+    - name: ExampleCustomRecognizer
+      patterns:
+        - name: "zip code (weak)"
+          regex: "(\\b\\d{5}(?:\\-\\d{4})?\\b)"
+          score: 0.01
+        - name: "zip code (weak)"
+          regex: "(\\b\\d{5}(?:\\-\\d{4})?\\b)"
+          score: 0.01
+      supported_languages:
+        - language: es
+          context: [tarjeta,credito]
+        - language: en
+          context: [credit]
+      supported_entity: "ZIP"
+      deny_list: [Mr., Mrs., Ms., Miss, Dr., Prof.]
```

Comment:
deny list fits the TitlesRecognizer and demonstrated there, can be dropped?

Reply:
Agree, we can perhaps have two examples, one for deny-lists (titles) and one for zip code (regex)

Reply:
removed